names=new
visibility=public
kind=defined

--- new(error_message = "", name = nil, args = nil, priv = false) -> NoMethodError

例外オブジェクトを生成して返します。

@param error_message エラーメッセージを表す文字列です

@param name 未定義だったシンボルです

@param args メソッド呼び出しに使われた引数です

@param priv private なメソッドを呼び出せる形式 (関数形式(レシーバを省略した形式)) で呼ばれたかどうかを指定します

例:

  nom = NoMethodError.new("message", "foo", [1,2,3])
  p nom.name
  p nom.args
  
  # => "foo"
  [1, 2, 3]

